### DISCUSSION TITLE
Hi.. Sorry to Bother You Guys But I'm Stuck..

### DISCUSSION THREAD

**Discussion by RiSK907** [nefarius/DsHidMini#305] [Support]
I downloaded this program and was so excited to find one like it. But once I unzipped the program, I couldn't find something to install/a .exe to open the program...? I am faced with x64 and a x86 files and I click on the x64 folder, as it meets my computer's standards. But after that, I'm at a dead end... just around 7 or 6 files that don't seem to move this process forward (then again, I could just be an idiot). I very much apologize if I'm simply stupid and am  missing something obvious here. If anyone could help me, I'd greatly appreciate it =____= 

Thank you!!!

**Comment by nefarius** [comment]
The installation instructions are literally linked multiple times on each release page, follow them, thousands of people succeeded before you 😉
